### PR TITLE
BZ-18845: feat: select component

### PR DESCRIPTION
**Pull request by anjalideswal**
- added showDropDownArrow optional property to toggle arrow visbility
- added dynamic css variables for box-shadow, hover background-color, selected-option-icon

Dev proof - https://drive.google.com/file/d/1QSu0lkBobQh6YFMqDltyLcWt_vN_lZIG/view?usp=sharing

### CODE REVIEW

**Review by vishalbhardwaj7 — APPROVED** [review]

**Review by rab1prasad — APPROVED** [review]

File: src/lib/Select/Select.svelte [hunk 1]
```diff
@@ -155,11 +155,13 @@
...
       {/if}
       <div class="filler" />
-      <img
-        src="https://sdk.breeze.in/gallery/icons/right-arrow.svg"
-        alt=""
-        class="arrow {isSelectOpen ? 'active' : ''}"
-      />
+      {#if properties.showDropDownArrow}
+        <img
+          src="https://sdk.breeze.in/gallery/icons/right-arrow.svg"
```
Comment (murdore) [inline src/lib/Select/Select.svelte]: can we pass this as input? also expose the alt tag to be passed by component user

**Review by murdore — CHANGES_REQUESTED** [review]

**Comment by sinha-sahil** [comment]
@anjalideswal This seems to be more of a enhancement than a fix. Kindly, update the type.

File: src/lib/Select/Select.svelte (hunk 1 above)
Comment (anjalideswal) [reply src/lib/Select/Select.svelte]: Done. Added `dropDownIcon` as optional prop. If not passed, drop down icon will not be added, to handle drop down icon visibility. Added altText export

File: src/lib/Select/Select.svelte [hunk 2]
```diff
@@ -5,6 +5,7 @@
   import Button from '$lib/Button/Button.svelte';
 
   let selectedElementDiv: HTMLDivElement | null = null;
+  export let altText = '';
```
Comment (rab1prasad) [inline src/lib/Select/Select.svelte]: rename, please specify for whom this alt tex is for.

File: src/lib/Select/properties.ts [hunk 3]
```diff
@@ -6,4 +6,5 @@ export type SelectProperties = {
   selectedItemLabel: string | string[] | null;
   showSelectedItemInDropdown: boolean;
   selectMultipleItems: boolean;
+  dropDownIcon?: string | null;
```
Comment (rab1prasad) [inline src/lib/Select/properties.ts]: Do we need this `?` if using null?

I think we should have 2 different props one for `showDropDownIcon` one for dropdown icon, and a default icon should be provided if icon is not passed as prop.

**Review by murdore — APPROVED** [review]

File: src/lib/Select/Select.svelte [hunk 4]
```diff
@@ -155,11 +158,13 @@
         {properties.selectedItemLabel}
       {/if}
       <div class="filler" />
-      <img
-        src="https://sdk.breeze.in/gallery/icons/right-arrow.svg"
-        alt=""
-        class="arrow {isSelectOpen ? 'active' : ''}"
-      />
+      {#if !properties.hideDropDownIcon}
+        <img
```
Comment (murdore) [inline src/lib/Select/Select.svelte:173]: @anjalideswal lets use the Image component from svelteComponents.
Check @adarshjuspay 's pr for reference

**Review by rab1prasad — APPROVED** [review]

**Review by kingk5 — APPROVED** [review]

**Comment by murdore** [comment]
@anjalideswal resolve conflicts and add proof of testing

**Review by murdore — APPROVED** [review]

**Pull request merged**